### PR TITLE
Iterate viwiki

### PR DESCRIPTION
**Pull request by HAKSOAT**

### CODE REVIEW

File: model_info/viwiki.articletopic.md [hunk 1]
```diff
@@ -554,408 +554,408 @@ Model Information:
...
+		STEM.Engineering                             0.983
+		STEM.Libraries & Information                 0.964
+		STEM.Mathematics                             0.984
+		STEM.Medicine & Health                       0.976
+		STEM.Physics                                 0.987
+		STEM.STEM*                                   0.982
+		STEM.Space                                   0.99
+		STEM.Technology                              0.977
 		-------------------------------------------  -----
-	pr_auc (micro=0.826, macro=0.684):
```
Comment (HAKSOAT) [inline model_info/viwiki.articletopic.md:892]: @halfak Macro is higher, but micro is lower at cutoff 20000.

The values at cutoff 10000 were:

pr_auc (micro=0.711, macro=0.533)

File: model_info/viwiki.articletopic.md [hunk 2]
```diff
@@ -554,408 +554,408 @@ Model Information:
...
+	pr_auc (micro=0.706, macro=0.541):
```
Comment (HAKSOAT) [inline model_info/viwiki.articletopic.md:892]: @halfak Macro is higher, but micro is lower at cutoff 20000.

The values at cutoff 10000 were:

pr_auc (micro=0.711, macro=0.533)

Comment (halfak) [reply model_info/viwiki.articletopic.md:892]: That's really strange!  I wonder if something weird is happening.  Can we replicate the stats we got using the old 100k vocab?

Comment (HAKSOAT) [reply model_info/viwiki.articletopic.md:892]: "Can we replicate the stats we got using the old 100k vocab?"

I don't really get this

Comment (halfak) [reply model_info/viwiki.articletopic.md:892]: I wonder if we'd get comparable model statistics if we were to revert your changes and rebuild the model.  If we can't, we know that something else is under-cutting the fitness of the model.  If we can, then maybe something in the quantization process that isn't working correctly.

File: model_info/viwiki.articletopic.md [hunk 3]
```diff
@@ -816,146 +816,146 @@ Model Information:
...
 		STEM.Physics                                 0.987
-		STEM.STEM*                                   0.982
-		STEM.Space                                   0.99
-		STEM.Technology                              0.977
+		STEM.STEM*                                   0.98
+		STEM.Space                                   0.989
+		STEM.Technology                              0.975
 		-------------------------------------------  -----
-	pr_auc (micro=0.706, macro=0.541):
+	pr_auc (micro=0.703, macro=0.534):
```
Comment (halfak) [inline model_info/viwiki.articletopic.md:892]: This makes it clear that it's the new vectors that are undercutting our fitness -- not something else in the pipeline.  I'm working on rebuilding the viwiki vectors to make sure nothing went strangely there.  I'll have a fresh 10k vector set to try some time today (July 13th)

Comment (HAKSOAT) [reply model_info/viwiki.articletopic.md]: Improvements :-D 

pr_auc (micro=0.82, macro=0.672)

File: model_info/viwiki.articletopic.md (hunk 2 above)
Comment (HAKSOAT) [reply model_info/viwiki.articletopic.md]: But slight drop compared to the original

Original:
pr_auc (micro=0.826, macro=0.684)

New:

pr_auc (micro=0.82, macro=0.672)

**Pull request merged**